docs/lineage/manual-lineage-feature-guide.md

```diff
@@ -0,0 +1,123 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Manual Lineage
+
+<FeatureAvailability/>
+
+Starting in version `0.9.5`, DataHub supports the manual editing of lineage between entities in the UI. Add or remove upstream and downstream lineage edges in both the Lineage Visualization screen as well as the Lineage tab on entity pages. Use this feature to supplement automatic lineage extraction in ingestion or establish important entity relationships in sources that we don't support automatic extraction for yet! Manual lineage is only supported for Datasets, Charts, Dashboards, and Data Jobs for now.
+
+:::note
+
+If you edit lineage manually through the UI but have lineage automatically extracted in ingestion for the entities you edited, your edits from the UI may be overwritten when ingestion runs again.
```

Comment:
How about:

**Note**: Lineage added by hand and programmatically may conflict with one another to cause unwanted overwrites. We strongly recommend only editing lineage by hand if you are _not_ also extracting lineage automatically, e.g. by running an ingestion source.

Reply:
much nicer